Source/Resource/Help/Data Format/Data Format.html

```diff
@@ -118,7 +118,8 @@ <h3 style="color:blue">
                 Open a video file for evaluation and then select "Export &gt; To qctools.xml.gz".
             </li>
             <li>Via FFmpeg (version 2.3 or later)<br />
-                For a file with video and audio named EXAMPLE.mov<br />
+                
+                For a file with video and audio named EXAMPLE.mov (If your video does not contain audio streams, this command will result in an invalid argument)<br />
```

Comment:
I'm not sure where the change was, but in the latest ffmpeg release the presence of audio arguments with no audio stream does not cause an error.